api/v1alpha1/sandbox_types.go

```diff
@@ -21,9 +21,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	TTLFromCreate = "create"
+	TTLFromReady  = "ready"
+	TTLDisable    = "disable"
```

Comment:
When I saw `ttlFrom: disable` config, I thought it was trying to say "start TTL after the sandbox is disabled"... Could we make this option more readable? E.g. instead of saying "disable", we say something like "disableTTL".

Reply:
or what about `ttlFrom: never` ?

Reply:
I like `never`: Very descriptive and clear!

Reply:
done.